config/locales/client.en.yml

```diff
@@ -46,8 +46,11 @@ en:
       types:
         reviewable_akismet_post:
           title: "Akismet Flagged Post"
+          noun: "post"
```

Comment:
Curious, where is `noun` used?

Reply:
It is used to construct the context question, e.g.:

> Is this **post** spam.